auth0/src/main/java/com/auth0/android/authentication/storage/SecureCredentialsManager.kt

```diff
@@ -340,6 +343,10 @@ public class SecureCredentialsManager @VisibleForTesting(otherwise = VisibleForT
         if (scope != null) {
             request.addParameter("scope", scope)
         }
+        parameters.forEach { (key, value) ->
```

Comment:
all can be added at once. See https://github.com/auth0/Auth0.Android/blob/main/auth0/src/main/java/com/auth0/android/request/Request.kt#L35